docs/contribution-guidelines.md

```diff
@@ -28,72 +28,86 @@ Streamability V2 is attempting to have a linear commit history on `main` as well
 Creating and pushing a new feature branch is quite simple. Follow the steps below:
 
 1. Checkout to `develop` and pull the most recent changes.
-```s
+
+```sh
 git pull origin develop
 ```
 
 2. Create a new feature branch using the proper naming convention.
-```s
-git checkout -b <"issue-num"-"issue-type"-"branch-name">
+
+```sH
+git checkout -b <"issue-num"-"branch-name">
```

Comment:
I think this might be a little repitive because of the branching-strategy example above.